internal/config/parse.go

```diff
@@ -139,11 +138,6 @@ func (p Provider) Validate() error {
 		result = errors.Join(result, errors.New("must have a 'name' property"))
 	}
 
-	// All schema refs must be a local, file, or http resolve type
-	if p.SchemaRef != "" && index.DetermineReferenceResolveType(p.SchemaRef) < 0 {
-		result = errors.Join(result, errors.New("'schema_ref' must be a valid JSON schema reference"))
-	}
-
```

Comment:
This is the only removed functionality from upgrading. The `index` package in `pb33f/libopenapi` has been significantly refactored and I can't find an equivalent function for this.

In reality, this validation wasn't super useful anyways, since it never actually checked the spec for the existence of a reference, it just validates that the reference "could" be valid via the syntax.

If we want to replace this validation in the future, we should probably just load the OpenAPI spec provided and check if the reference exists in the spec.